src/components/combobox/combobox.e2e.ts

```diff
@@ -481,21 +471,188 @@ describe("calcite-combobox", () => {
     });
   });
 
+  describe("mouse navigation", () => {
+    let page: E2EPage;
+    let label;
+    let popper;
+
+    beforeEach(async () => {
+      page = await newE2EPage();
+      await page.setContent(
+        html`
+          <calcite-label id="label">
```

Comment:
Clicking on the label should behave the same as clicking on the component, so I think we could remove the `calcite-label` aspect from these tests. Label integration is already handled by the `labelable` helper.